docs/source/technical_design/localinstall.rst

```diff
@@ -107,6 +107,22 @@ Go to the directory containing openkat:
 	$ cd nl-kat-coordination
 	$ make update
 
+Create a new superuser for the new version. You can delete the old superuser after the update. This is not pretty, but has the advantage that your databases remain intact. Check that you are on the most recent version everywhere, especially Rocky sometimes hangs because of yarn.lock.
```

Comment:
Why do you need to create a new superuser each time you update?

Reply:
This is a merge problem, this paragraph was removed last week in https://github.com/minvws/nl-kat-coordination/commit/5a2315e29ba20488b7a2bae3049444615cf52539#diff-174ecb96956e599b0bfe65c2f1d191c2212fdcb5a8bff742178fcd2e45d1b189 but somehow got back in with this PR.

Reply:
Removed this paragraph like intended.